services/scabbard/libscabbard/src/store/scabbard_store/diesel/models.rs

```diff
@@ -602,6 +621,10 @@ impl TryFrom<(&Context, &i64)> for UpdateContextActionParticipantList {
     fn try_from((context, action_id): (&Context, &i64)) -> Result<Self, Self::Error> {
         let mut participants = Vec::new();
         for participant in context.participants() {
+            let decision_ack = match participant.decision_ack {
+                true => "TRUE".to_string(),
```

Comment:
Given this value is always one of the two, it could just be `&'static str`

Reply:
switched to an enum